Code/Editor/Plugins/ProjectSettingsTool/ProjectSettingsContainer.cpp

```diff
@@ -282,35 +311,49 @@ namespace ProjectSettingsTool
         return m_projectJson.m_document->GetAllocator();
     }
 
-    const char* ProjectSettingsContainer::GetFailedLoadingPlistText()
-    {   
-        return "Failed to load info.plist"; 
+    void ProjectSettingsContainer::LoadJson(JsonSettings& jsonSettings)
+    {
+        StringOutcome outcome = ReadConfigFile(jsonSettings.m_path, jsonSettings.m_rawData);
+        if (!outcome.IsSuccess())
+        {
+            m_errors.push(SettingsError(AZStd::string::format("Failed to load %s", jsonSettings.m_path.c_str()), outcome.GetError(), true /*shouldAbort*/));
+        }
+
+        jsonSettings.m_document->Parse(jsonSettings.m_rawData.c_str());
     }
 
-    void ProjectSettingsContainer::LoadProjectJsonData()
+    void ProjectSettingsContainer::SaveJson(const JsonSettings& jsonSettings)
     {
-        StringOutcome outcome = ReadConfigFile(m_projectJson.m_path, m_projectJson.m_rawData);
+        // Needed to write a document out to a string
+        rapidjson::StringBuffer jsonDataBuffer;
+        // Use pretty writer so it can be read easier
+        rapidjson::PrettyWriter<rapidjson::StringBuffer> jsonDatawriter(jsonDataBuffer);
+
+        jsonSettings.m_document->Accept(jsonDatawriter);
+        const AZStd::string jsonDataString = jsonDataBuffer.GetString();
+
+        StringOutcome outcome = WriteConfigFile(jsonSettings.m_path, jsonDataString);
+
         if (!outcome.IsSuccess())
         {
-            m_errors.push(SettingsError("Failed to load project.json", outcome.GetError()));
+            m_errors.push(SettingsError(AZStd::string::format("Failed to save %s", jsonSettings.m_path.c_str()), outcome.GetError()));
         }
-
-        m_projectJson.m_document->Parse(m_projectJson.m_rawData.c_str());
     }
 
-    // Loads info.plist for iOS from disk
     void ProjectSettingsContainer::LoadPlist(PlistSettings& plistSettings)
     {
         StringOutcome outcome = ReadConfigFile(plistSettings.m_path, plistSettings.m_rawData);
         if (!outcome.IsSuccess())
         {
-            m_errors.push(SettingsError(GetFailedLoadingPlistText(), outcome.GetError()));
+            m_errors.push(SettingsError(AZStd::string::format("Failed to load %s", plistSettings.m_path.c_str()), outcome.GetError(), true /*shouldAbort*/));
         }
 
+        static const int xmlFlags = rapidxml::parse_doctype_node | rapidxml::parse_declaration_node | rapidxml::parse_no_data_nodes;
```

Comment:
nit: Likely unnecessary to add `static` here in the body of the function, keeping as `const int` is (and maybe even making the name start with an upper case (`XmlFlags`) might be clearer